drivers/sensor/si1133/Kconfig

```diff
@@ -0,0 +1,28 @@
+# Si1133 UV Index and Ambient Light Sensor configuration options
+
+# Copyright (c) 2021 Murali Tejeshwar Janaswami
+# SPDX-License-Identifier: Apache-2.0
+
+config SI1133
+	bool "Si1133 UV Index and Ambient Light Sensor"
+	depends on I2C
+	help
+	  Enable driver for Si1133 UV Index and Ambient Light Sensor.
+
+if SI1133
+
+choice SI1133_OPERATING_MODE
```

Comment:
Why is this choice needed when the reference driver from mbed works fine (as far as I can tell) across all light levels?

Reply:
Sunlight has much higher lux values than indoor lighting. 16-bit unsigned variables will not be sufficient for holding values larger than 65535. This operating mode determines if the sensor is to be operated in 24-bit mode(which can store larger values) or 16-bit mode(which is sufficient for low light scenarios).

Reply:
Right, but the reference driver configures 2 photodiodes and chooses which one to use for the output based on the saturation of the more sensitive photodiode. Why can't we do the same thing here?